lib/theme/widgets/hs_warning_bottom_sheet.dart

```diff
@@ -0,0 +1,112 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/flutter_svg.dart';
+import 'package:hatspace/dimens/hs_dimens.dart';
+import 'package:hatspace/theme/hs_theme.dart';
+import 'package:hatspace/theme/widgets/hs_buttons.dart';
+
+class HsWarningBottomSheetView extends StatelessWidget {
+  final String? iconUrl;
+  final String? title;
+  final String? description;
+  final String? primaryButtonLabel; // return false value
+  final VoidCallback? primaryOnPressed;
+  final String? secondaryButtonLabel; // return true value
+  final VoidCallback? secondaryOnPressed;
```

Comment:
We need at least 1 button

Reply:
Which button will be required? With some bottom sheets, it only needs a negative button. The secondary button should be displayed, or the tertiary button. We don't know which button will be displayed. That's why making all the buttons is optional. Making them optional, which is not that no buttons will be displayed.